frontend/packages/console-plugin-sdk/src/typings/kebab-actions.ts

```diff
@@ -0,0 +1,18 @@
+import { K8sKind } from '@console/internal/module/k8s';
+import { KebabAction } from '@console/internal/components/utils/kebab';
+import { Extension } from './extension';
+
+namespace ExtensionProperties {
+  export interface PluginKebabActions {
+    // no kind should return any actions common for all kinds
+    getKebabActionsForKind: (kind: K8sKind) => KebabAction[];
+  }
+}
+
+export interface KebabActions extends Extension<ExtensionProperties.PluginKebabActions> {
```

Comment:
Yeah, I would too. Kept this way since the `KebabAction`  type from console is used. As commented below, when we update to have the type defs in the SDK it can be re-evaluated.